src/collection-facets/more-facets-content.ts

```diff
@@ -195,32 +229,24 @@ export class MoreFacetsContent extends LitElement {
   /**
    * Combines the selected facets with the aggregations to create a single list of facets
    */
-  private get mergedFacets(): FacetGroup[] | void {
```

Comment:
It seems like a great deal of this logic was originally copied over from `collection-facets.ts`, but wasn't fully adjusted to account for the differences (e.g., More Facets only ever showing a single facet group, instead of multiple at once).

I've aimed to simplify the logic around these getters to (1) eliminate a bunch of unnecessary looping over aggregation types that aren't relevant to the one being displayed, and (2) return only the relevant facet group, rather than an array of groups where only one is useful.